app/scripts/services/browserStorage.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BrowserStorage', function() {
+
+    // Automaticaly namespaces all of our storage, wether in session or local
+    var namespace = 'openshift-';
+
+    function Store(type, key) {
```

Comment:
Thats true, mine is a bit unique as I'm storing a map rather than one value.
I can roll this back & update to support either use case in a follow-on.